tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -53,6 +54,24 @@ def run(ceph_cluster, **kwargs) -> int:
         executor = RadosBench(mon_node=client, clients=clients)
 
     try:
+        if rhbuild.startswith("8"):
```

Comment:
I would suggest to keep this outside the test modules, so it would be easy to use or remove anytime. and please make it re-usable, call it like a class with all necessary logging.
```
class BZ2314146:

      def balancer_off(self):
      ...
      def ...
      ...
```

Reply:
Hi @sunilkumarn417 
The current code changes have been done keeping in mind that this workaround would only be around for next few weeks.
Had this been a permanent code change or a significant one, it would make sense to put in the effort to modularize the code.

Even we make it re-usable, we would still need to raise another PR to remove it from the test case modules when the fix arrives.

Please let me know if you'd still like a re-usable class to be created, I will remove the modified files other than rados upgrade test module from this PR and respective owners take a call on how they would like to include the workaround in their test case modules.